added logfile viewing capabilities

rdoshi99 opened the pull request:
Addresses #83 

- adds logfile nav table as a drop down feature under summary
![image](https://user-images.githubusercontent.com/46800082/60307151-37fab780-98f8-11e9-819c-faf3a816a912.png)

- a single log file looks like:
![image](https://user-images.githubusercontent.com/46800082/60307161-43e67980-98f8-11e9-9493-c67bed669e68.png)


TODO: not sure how to add the highlights

rdoshi99 commented:
Added highlights for errors and warnings, formatted the logfile:
![image](https://user-images.githubusercontent.com/46800082/60308752-3385cd00-98ff-11e9-9d82-7d15b3a8710e.png)


![image](https://user-images.githubusercontent.com/46800082/60308755-37195400-98ff-11e9-9c9c-8ccab0f02b47.png)

sbailey commented on py/qqa/webpages/templates/logfile.html:
```diff
@@ -0,0 +1,59 @@
+.flex-item{
+  flex-grow: 1;
+}
+</style>
```
Are HTML files allowed to have multiple <style> definition blocks?  Are they allowed in the <body>?  `base.html` also has a style block between the <head> and the <body>.  Maybe this is perfectly fine, or maybe it violates the standard and browsers are just being forgiving.  Please check.

rdoshi99 replied:
It extends from both, but I think some restructuring will be good anyway. The preproc image nav tables also use the same style options that I copy-pasted, so perhaps a parent template can serve all of them

sbailey commented on py/qqa/webpages/templates/logtable.html:
```diff
@@ -0,0 +1,75 @@
+  font-weight:bold;
+  font-size: 25px;
+}
+</style>
```
logtable.html and logfile.html have duplicated <style> blocks.  Could you structure this so that they inherit a common style?  We must be bumping into one of the organizational reasons that CSS are normally separate files and not embedded into the HTML too.  Consider what the right solution is in this case (I'm not sure).

rdoshi99 commented:
Addressed above comments: 
![image](https://user-images.githubusercontent.com/46800082/60374093-8b800a80-99b7-11e9-8d01-c6cf42a4599f.png)

![image](https://user-images.githubusercontent.com/46800082/60374096-9175eb80-99b7-11e9-8787-5f909dc87e1d.png)

And refactored templates so that the logfile and logtable pages are generated from the same template, and a similar restructuring for the plotimage and preproc-nav files

rdoshi99 commented:
This shows the spectro-cam nav table to scroll with the page, so users don't have to jump back to the top to reach it.
![image](https://user-images.githubusercontent.com/46800082/60374096-9175eb80-99b7-11e9-8787-5f909dc87e1d.png)
